ngrinder-controller/build.gradle

```diff
@@ -2,48 +2,49 @@ description = "ngrinder-controller"
         maven { url "https://repo.spring.io/libs-milestone" }
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.0.6.RELEASE")
     }
-}
-
-configurations {
-    executableWarDeps
 }
 
 apply plugin: "war"
+apply plugin: "org.springframework.boot"
 
 sourceSets.main.output.resourcesDir = sourceSets.main.output.classesDir
 
+def agentScriptPath = "${sourceSets.main.output.resourcesDir}/ngrinder-sh/agent"
+def monitorScriptPath = "${sourceSets.main.output.resourcesDir}/ngrinder-sh/monitor"
+
 dependencies {
-    compile project(":ngrinder-starter")
     compile project(":ngrinder-core")
-    compile project(":ngrinder-sh")
     compile files("${rootDir}/lib/universal-analytics-java-1.0.jar")
+
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-freemarker', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-loader', version: springboot_version)
+
     compile (group: "rome", name: "rome", version: "1.0")
     compile (group: "com.ibm.icu", name: "icu4j", version: "4.6")
     compile (group: "sonia.svnkit", name: "svnkit-dav", version: svnkit_version)
     compile (group: "sonia.svnkit", name: "svnkit", version: svnkit_version)
     compile (group: "javax.servlet.jsp", name: "jsp-api", version: "2.1")
     compile (group: "org.python", name: "jython-standalone", version: "2.5.3")
-    compile (group: "org.freemarker", name: "freemarker", version: "2.3.20")
     compile (group: "net.sf.ehcache", name: "ehcache-core", version: "2.5.1")
     compile (group: "com.google.guava", name: "guava", version: "12.0")
-    compile (group: "org.springframework", name: "spring-web", version: spring_version)
-    compile (group: "org.springframework", name: "spring-tx", version: spring_version)
-    compile (group: "org.springframework", name: "spring-context-support", version: spring_version)
-    compile (group: "org.springframework", name: "spring-aspects", version: spring_version)
-    compile (group: "org.springframework", name: "spring-expression", version: spring_version)
     compile (group: "org.springframework.security", name: "spring-security-web", version: spring_security_version)
     compile (group: "org.springframework.security", name: "spring-security-taglibs", version: spring_security_version)
     compile (group: "org.springframework.security", name: "spring-security-config", version: spring_security_version)
```

Comment:
Do we need above three lines? Arent' they automatically imported but spring-boot-starter-security?

Reply:
web, config is auto imported by spring-boot-starter-security. but taglibs is not.

so taglibs need to declare explicitly.

Reply:
ok. just leave taglibs but remove the others.